float64[] action
float64[] rpy

---
float64[] joint_pos
float64[] state
float64[] joint_angle
float64[] limit
float64[] quaterniond
bool      quat_inv
bool      success
